src/components/generic/Modal/Modal.scss

```diff
@@ -1,5 +1,34 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-modal {
-  .md-modal__content {
-    padding: 2rem;
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .modal-content {
+    width: 500px;
+    height: 456px;
+    border: 1px solid var(--wxc-modal-border-color);
```

Comment:
maybe just --wxc-modal-border, so someone could change the style of border and width as well